6.bulk_Morphology_Elastic_Network/2.test_model/scripts/test_model_call_cluster.sh

```diff
@@ -19,7 +19,7 @@ readarray -t cytokine_array < $filename
 
 
 shuffles=( "True" "False" )
-cell_types=( SHSY5Y PBMC )
+cell_types=( PBMC )
```

Comment:
Again, you might not want these to change... I'm not going to make these comments from here on out - please confirm next time you go through!